tests/test_new_meshing_workflow.py

```diff
@@ -1055,6 +1055,20 @@ def downstream_names(task):
     ]
 
 
+@pytest.mark.codegen_required
+@pytest.mark.fluent_version(">=23.2")
+def test_new_workflow_structure(new_mesh_session):
+    meshing = new_mesh_session
+    watertight = meshing.watertight()
+    assert watertight.import_geometry.arguments()
+    with pytest.raises(AttributeError) as msg:
+        watertight.TaskObject["Import Geometry"]
+    assert (
+        msg.value.args[0]
+        == "'WatertightMeshingWorkflow' object has no attribute 'TaskObject'"
```

Comment:
don't really care about the message enough to hard-code it in the test

Reply:
As mentioned previously, `name` and `obj` attributes were added in 3.10, so when 3.10 becomes are baseline version, then we can test in terms of those, but before that point, I would prefer to leave out this kind of testing.

Reply:
There are a lot of tests using this mechanism. I'll clean them up as well as part of a separate clean-up PR to resolve this kind of issue at once.